ios/Podfile.lock

```diff
@@ -362,7 +362,7 @@ PODS:
     - React
   - RNReanimated (1.13.0):
     - React
-  - RNSentry (2.2.1):
+  - RNSentry (2.4.3):
```

Comment:
> We should have done this in 7dce4e7. It looks like it was missed
> because the quality-of-life improvement in 683f387 came with the
> side effect of skipping `pod install` even when autolinking wants to
> upgrade the Podfile.lock because of an NPM package upgrade. Hmm.

Any ideas/plans on how to fix this in general?

Reply:
Hmm. I think I'll revert 683f3875e for now, and ponder a better fix later.